### PR TITLE
sentry staging 2719 RAMSynchronizer fix

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: src/etools/applications/reports/synchronizers.py [hunk 1]
```diff
@@ -341,10 +341,10 @@ def _clean_records(self, records):
             a = r['WBS_ELEMENT_CODE']
             code = str(r['INDICATOR_CODE'])
             mapped_records[code] = {
-                'name': r['INDICATOR_DESCRIPTION'][:1024],
-                'baseline': r['INDICATOR_BASELINE'][:255],
+                'name': str(r['INDICATOR_DESCRIPTION'])[:1024],
```
Comment (robertavram) [inline src/etools/applications/reports/synchronizers.py:344]: set to empty string or None rather then the string "None" as there's a filter on line331

File: src/etools/applications/reports/synchronizers.py [hunk 2]
```diff
@@ -341,10 +341,10 @@ def _clean_records(self, records):
...
+                'baseline': str(r['INDICATOR_BASELINE'])[:255],
                 'code': code,
-                'target': r['INDICATOR_TARGET'][:255],
+                'target': str(r['INDICATOR_TARGET'])[:255],
```
Comment (robertavram) [inline src/etools/applications/reports/synchronizers.py:347]: an indicator without a target is meaningless, I would suggest to use the same logic as for the name and filter these out.

**Review by robertavram — CHANGES_REQUESTED** [review]

File: src/etools/applications/reports/synchronizers.py (hunk 1 above)
Comment (domdinicola) [reply src/etools/applications/reports/synchronizers.py:344]: this was @ntrncic input

Comment (ntrncic) [reply src/etools/applications/reports/synchronizers.py]: I meant only for the baseline. @domdinicola

**Pull request merged**